### PR TITLE
Readme Update

### PR DESCRIPTION
**Pull request by djbritt**

### CODE REVIEW

**Comment by djbritt** [comment]
Sorry, kinda new to pull requests. I did three commit's on accident. But the last commit is correct! I also didn't know how to reduce the amount of commits to do a pull request on.

File: README.md [hunk 1]
```diff
@@ -35,23 +35,24 @@ You can also see this component in action at [voicerecordpro.com](https://www.vo
...
 
-class Example extends React.Component {
+export class Example extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       record: false
-    }
+    };
+    this.onClick = this.startRecording.bind(this);
```
Comment (hackingbeauty) [inline README.md]: Hi,

startRecording and stopRecording are defined using  es6 "fat" arrow functions.  Therefore you don't need to bind them to the Example class in the constructor.  The "this" keyword will have the correct scope when startRecording & stopRecording are defined this way:

```
startRecording= () => { }
stopRecording= () => { }
```

Can you remove these lines in your PR?

**Comment by djbritt** [comment]
I also am not a pro at React, but I the changes I made in the readme are the changes I made to my react app, and it worked. It took me about an hour to figure out the changes.

**Comment by djbritt** [comment]
Hey, so the changes that got merged weren't correct for some reason.. 

Check out the readme changes I made on my fork
https://github.com/djbritt/react-mic

**Comment by hackingbeauty** [comment]
OK, I manually updated the README.  Thanks for catching these issues!

**Comment by subtopic** [comment]
Any idea why the pull request didn't work when I did it? This is my first pull request.

**Comment by hackingbeauty** [comment]
@subtopic, I only received a PR from @djbritt.  Are you sure you issued a PR?

**Comment by djbritt** [comment]
Ah shoot... Sorry, wrong account. But here is my current repo

https://github.com/hackingbeauty/react-mic

I did a pull request from the account, and you can see the changes on the README.md file. Any idea why the PR wouldn't go through?

**Comment by hackingbeauty** [comment]
No, but you can try again.  Just make a small change and I'll merge the PR.

**Comment by djbritt** [comment]
Thank you! I really appreciate it.

**Pull request merged**